pkg/api/v1/types.go

```diff
@@ -533,6 +533,19 @@ func NewVMIReferenceFromNameWithNS(namespace string, name string) *VirtualMachin
 	return vmi
 }
 
+func NewVMIReferenceWithLabels(nodeName string, name string) *VirtualMachineInstance {
```

Comment:
This is just a helper for tests. Can you move it to the tests?

Reply:
Done